x-pack/plugin/sql/qa/src/main/resources/datetime-interval.csv-spec

```diff
@@ -20,119 +20,177 @@ SELECT INTERVAL 1 YEARS AS y, INTERVAL 2 MONTHS AS m, INTERVAL 3 DAYS AS d, INTE
 ;
 
 // take the examples from https://docs.microsoft.com/en-us/sql/odbc/reference/appendixes/interval-literals?view=sql-server-2017
-testYear
+year
 SELECT INTERVAL '326' YEAR;
 
 INTERVAL '326' YEAR
 -------------------
 +326-0 
 ;
 
-testMonth
+month
 SELECT INTERVAL '326' MONTH;
 
 INTERVAL '326' MONTH
 --------------------
 +0-326
 ;
 
-testDay
+day
 SELECT INTERVAL '3261' DAY;
 
 INTERVAL '3261' DAY
 -------------------
 +3261 00:00:00.0 
 ;
 
-testHour
+hour
 SELECT INTERVAL '163' HOUR;
 
 INTERVAL '163' HOUR
 -------------------
 +6 19:00:00.0  
 ;
 
-testMinute
+minute
 SELECT INTERVAL '163' MINUTE;
 
 INTERVAL '163' MINUTE
 ---------------------
 +0 02:43:00.0
 ;
 
-testSecond
+second
 SELECT INTERVAL '223.16' SECOND;
 
 INTERVAL '223.16' SECOND
 ------------------------
 +0 00:03:43.16  
 ;
 
-testYearMonth
+yearMonth
 SELECT INTERVAL '163-11' YEAR TO MONTH;
 
 INTERVAL '163-11' YEAR TO MONTH
 -------------------------------
 +163-11  
 ;
 
-testDayHour
+dayHour
 SELECT INTERVAL '163 12' DAY TO HOUR;
 
 INTERVAL '163 12' DAY TO HOUR
 -----------------------------
 +163 12:00:00.0 
 ;
 
-testDayMinute
+dayMinute
 SELECT INTERVAL '163 12:39' DAY TO MINUTE AS interval;
 
 interval
 ---------------
 +163 12:39:00.0 
 ;
 
-testDaySecond
+daySecond
 SELECT INTERVAL '163 12:39:59.163' DAY TO SECOND AS interval;
 
 interval
 ---------------
 +163 12:39:59.163  
 ;
 
-testDaySecondNegative
+daySecondNegative
 SELECT INTERVAL -'163 23:39:56.23' DAY TO SECOND AS interval;
 
 interval
 ---------------
 -163 23:39:56.23    
 ;
 
-testHourMinute
+hourMinute
 SELECT INTERVAL '163:39' HOUR TO MINUTE AS interval;
 
 interval
 ---------------
 +6 19:39:00.0
 ;
 
-testHourSecond
+hourSecond
 SELECT INTERVAL '163:39:59.163' HOUR TO SECOND AS interval;
 
 interval
 ---------------
 +6 19:39:59.163   
 ;
 
-testMinuteSecond
+minuteSecond
 SELECT INTERVAL '163:59.163' MINUTE TO SECOND AS interval;
 
 interval
 ---------------
 +0 02:43:59.163  
 ;
 
-testDatePlusInterval
+intervalPlusInterval
+SELECT INTERVAL 1 DAY + INTERVAL 53 MINUTES;
+
+INTERVAL 1 DAY + INTERVAL 53 MINUTES
+------------------------------------
++1 00:53:00.0    
+;
+
+datePlusIntervalInline
```

Comment:
In this csv, `constant` is not used since for intervals that means repeating for every test so it becomes noise. Considering it has its own namespace (dedicated file) I think it's fine as is.